index.bs

```diff
@@ -923,10 +955,26 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Assert: |sourceToAttribute|'s [=attribution source/attribution mode=][0] is
     "`truthfully`" or "`never`".
 1. If the result of running
-    [=match an attribution source's filter data against an attribution trigger=] with
-    |sourceToAttribute| and |trigger| is false, return.
-1. If |trigger|'s [=attribution trigger/dedup key=] is not null and |sourceToAttribute|'s
-    [=attribution source/dedup keys=] [=list/contains=] it, return.
+    [=match an attribution source's filter data against filters=] with
+    |sourceToAttribute| and |trigger|'s [=attribution trigger/filters=] is false, return.
+1. Let |matchedConfig| be a new [=event-level trigger configuration=] with the items:
```

Comment:
I'm wondering if this default config is necessary. Maybe out of scope for this change but it seems like if none of the configs match we should not create a report.

cc @johnivdel

Reply:
That would be a behavioral change from https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-attribution-filters and the current implementation. We have top-level filters that exhibit that behavior, though.

Reply:
Ah, you're right. I think we need this if we want the filters to be an optional thing. Sorry for the noise!